qase-playwright/changelog.md

```diff
@@ -1,3 +1,17 @@
+# playwright-qase-reporter@2.0.3
+
+## What's new
+
+Added new annotation `qase.suite()`.
+Tests marked with it will be grouped in the Qase report by the specified suite name.
+
+```js
+test('test', async ({ page }) => {
+  qase.suite("Custom suite");
+  await page.goto('https://example.com');
+});
```

Comment:
Not for this PR, just a general question.

Can we declare our own `test` object (overriding one from the Playwright), define things like the root suite in it, and then use it without changing the test code?